.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -317,6 +325,14 @@ dynamodb_BatchWriteItem:
             - description:
               snippet_tags:
                 - ruby.example_code.dynamodb.BatchWriteItem
+    Swift:
+      versions:
+        - sdk_version: 1
+          github: swift/example_code/ddb/basics

```

Comment:
This should be swift/example_code/ddb -- point to the service folder, not the scenario.